src/parser.jl

```diff
@@ -130,7 +130,12 @@ end
 curline(ts::TokenStream)  = ts.lineno
 filename(ts::TokenStream) = symbol("none")
 
-line_number_node(ts) = LineNumberNode(curline(ts))
+if VERSION < v"0.4-dev+7034"
+    line_number_node(f, n) = LineNumberNode(n)
```

Comment:
When fixing the other errors I eventually got the same error that Lexicon had: `ERROR: LoadError: LoadError: MethodError: `convert` has no method matching convert(::Type{LineNumberNode}, ::Int64)
` I searched for it in Google, found Docile commit, tried that and it fixed the issue here.